src/index.ts

```diff
@@ -58,10 +44,33 @@ export default async function watchAndNotify(mongoData: MongoDataType, rabbitDat
     mongoConnection(mongoData, rabbitData, options);
 }
 
+
 function sleep(ms: number) {
     return new Promise(resolve => setTimeout(resolve, ms));
 }
 
+/**
+ * Get rabbit health status
+ * @returns {boolean} - true if healthy
```

Comment:
This seems like it returns a boolean inside of an object :confused:

Reply:
it's a known convention